src/utils/response.ts

```diff
@@ -0,0 +1,32 @@
+import { getTrackerMode, TrackerMode } from "./env_helper";
+
+export enum TrackerResponseType {
+  Success = "success",
+  Failed = "failed",
+}
+
+export class TrackerResponse {
+  status: number = 200;
+  state: TrackerResponseType = TrackerResponseType.Success;
```

Comment:
do we need state and status? can we assume 200 status is success (maybe with message set to "success") and anything else is an error?

Reply:
yeah, we only need one or the other. thanks again for asking these questions. i want to tighten this down. which one do you prefer?

Reply:
i think status 200 is good